src/config/config.py

```diff
@@ -36,6 +36,9 @@
 SUPPORTED_DISTROS = {
 	'IBM z/OS ™': {
 		'ZOS Software': 'ZOS_Software_List.json'
+	},
+	'Debian': {
+		'Buster': 'Debian_Buster_List.json'
```

Comment:
Strictly speaking, Buster is the code name for development, so we probably want to refer to it by the official version number, this is how the Debian project refers to it: https://www.debian.org/News/2019/20190706

Reply:
Debian 10 "buster"

Reply:
Ohh cool I'll do that